src/turing_inference.jl

```diff
@@ -1,16 +1,23 @@
 function turing_inference(prob::DiffEqBase.DEProblem,alg,t,data,priors = nothing;
+  likelihood_dist_priors = [InverseGamma(2, 3)],
+  likelihood = (u,p,t,σ) -> MvNormal(u, σ[1]*ones(length(u))),
```

Comment:
Only using ` σ[1]` here so why is  `σ` made a vector below?

Reply:
Because that's just the default